controllers/replica_set_controller.go

```diff
@@ -60,7 +60,7 @@ func init() {
 	zap.ReplaceGlobals(logger)
 }
 
-func NewReconciler(mgr manager.Manager) *ReplicaSetReconciler {
+func NewReconciler(mgr manager.Manager, mongodbRepoUrl, mongodbImage, mongodbImageType, agentImage, versionUpgradeHookImage, readinessProbeImage string) *ReplicaSetReconciler {
```

Comment:
What do you think of creating a structure to hold the controller configuration like you did in MEKO ?

Reply:
@Julien-Ben in MEKO `imageUrls` struct was necessary because we had to handle `RELATED_IMAGE_*` env vars which are not a static set. In here we have a set of images which are known in advance - I think it is a bit easier to follow without a map (without `imageUrls`).

But we might want to reconsider later.